app.py

```diff
@@ -59,54 +76,75 @@
 request_timeout = 2
 
 # Request only stable snaps
-snap_details_url = (
-    "https://api.snapcraft.io/api/v1/snaps/details/{snap_name}"
-    "?channel=stable"
-)
+snap_details_url = ''.join([
```

Comment:
There are a mix of lowercase constant variables and uppercase. I think we should go with just uppercase.
Also there are two sections of URL variables separated by the request session block. Can these URL blocks be put next to each other?